web/src/main/java/org/cbioportal/web/util/InvolvedCancerStudyExtractorInterceptor.java

```diff
@@ -127,7 +128,7 @@ public class InvolvedCancerStudyExtractorInterceptor extends HandlerInterceptorA
         if (!request.getMethod().equals("POST")) {
             return true; // no attribute extraction needed because all user supplied filter objects are in POST requests
         }
-        String requestPathInfo = request.getPathInfo();
+        String requestPathInfo = StringUtils.removeStart(request.getPathInfo(), "/column-store");
```

Comment:
or we can just do this if we want to avoid using a 3rd party utility class.

```suggestion
        String requestPathInfo = request.getPathInfo().replaceFirst("/column-store", "");
```